frontend/src/hooks/useFields.tsx

```diff
@@ -418,3 +428,57 @@ export function useFields<T extends Zaak = Zaak>(
     resetFilters,
   ];
 }
+
+const OverflowText = ({ text }: { text: string }) => {
+  const containerRef = useRef<HTMLDivElement>(null);
+  const [isOverflowing, setIsOverflowing] = useState(false);
+  const dialog = useDialog();
+
+  useEffect(() => {
+    const checkOverflow = () => {
+      if (containerRef.current) {
+        setIsOverflowing(
+          containerRef.current.scrollWidth > containerRef.current.clientWidth,
+        );
+      }
+    };
+
+    checkOverflow();
+    window.addEventListener("resize", checkOverflow);
```

Comment:
I don't think resize is the only time this may change, what about adding columns?